util/git/client.go

```diff
@@ -309,7 +311,16 @@ func (m *nativeGitClient) Checkout(revision string) error {
 // Otherwise, it returns an error indicating that the revision could not be resolved. This method
 // runs with in-memory storage and is safe to run concurrently, or to be run without a git
 // repository locally cloned.
-func (m *nativeGitClient) LsRemote(revision string) (string, error) {
+func (m *nativeGitClient) LsRemote(revision string) (res string, err error) {
+	for attempt := 0; attempt < m.retriesCount; attempt++ {
+		if res, err = m.lsRemote(revision); err == nil {
```

Comment:
thundering herd?

Reply:
I guess you are right. It is not safe to enable it by default. Changed the default number of attempts to 1.

In the next PR I'm going to document all performance-related env variables and recommendations of how to use them